### ISSUE TITLE
[TASK] Provide PowerPC CI Solutions Accessible for Newcomers

### ISSUE THREAD

**Issue by abebeos** [abebeos/PowerPC#7]
## Status

* Many OSS Projects do not have CI for PowerPC
* Setting up a CI from scratch can become a major barrier

## Solution Paths

* Provide an CI for a Project-Fork(!), document it (applicable to other projects)
    * Showcase to a project the simplicity of adding PowerPC CI 

Optional

* Setup CI on a remote machine
* Setup CI on a local machine

## Requirements

* Simplicity: zero-headache setup
* No need to be a core-developer (e.g.: a so called "power-user" could do the setup)
* Focus on ppc64le, keep other architectures in mind (e.g. IBM Z)

**Comment by edelsohn** [comment]
I agree that CI is important, but I don't understand what you are proposing.  PPC64LE Linux is present on Travis CI (although Travis CI new policies make this less attractive).  Also there is a public Jenkins instance at OSUOSL

https://osuosl.org/services/powerdev/request_powerci/

I agree that these are not publicized as well as they should be, but PowerPC CI is available.

**Comment by abebeos** [comment]
Estimated half of my efforts for the numba issue went to dev-system setup.

I want to reduce this effort, so anyone starting tasks on ppc does not spend time and energy for this. It is really a major barrier, and the delays for getting CI results are quite exhausting.

This one:

https://osuosl.org/services/powerdev/request_powerci/

is far too complex, and induces similar delays as the PowerPC instance that I got from... (don't remember, was a Brazilian domain). I wait 2 days. 2 days can be an eternity.

So, what I'm proposing is to makes this easy & faster (possibly even easier and faster than for x86)

1 hour, maximum, for anyone to get a PowerPC CI running is the goal (ideally on different CI systems, i would like to achieve this on azure pipelines, too, not sure if applicable easily)

**Comment by abebeos** [comment]
@edelsohn , I've updated the Issue-Text a little bit, should be clearer now.

**Comment by edelsohn** [comment]
I appreciate that a better CI for PowerPC would be good, but that is not in the scope of my project.  That cannot be a  prerequisite for enabling JITLink on PowerPC.  I am looking for a developer to enable JITLink and I am willing to negotiate a bounty for that.

**Comment by abebeos** [comment]
The prerequisites are not a "must-do", so all good.

**Comment by abebeos** [comment]
@edelsohn , for JITLink, please see #4
